src/components/ciPipeline/ciPipeline.service.ts

```diff
@@ -207,12 +206,8 @@ export function saveCIPipeline(
     isExternalCI,
     webhookConditionList,
     ciPipelineSourceTypeOptions,
-    imageTagValue?: string,
 ) {
     const ci = createCIPatchRequest(ciPipeline, formData, isExternalCI, webhookConditionList)
-    if(imageTagValue === ImageTagType.Default){
```

Comment:
is it intentional?

Reply:
Yes now we are just providing custom tag functionality